taxi/templatetags/query_transform.py

```diff
@@ -0,0 +1,16 @@
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag
+def query_transform(request, **kwargs):
+    print("request.GET", request.GET)
+    print("kwargs", kwargs)

```

Comment:
I guess we won't be needing prints in prod

Reply:
Thanks, changed.